gr-utils/python/modtool/cli/makeyaml.py

```diff
@@ -24,14 +24,21 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+import os
 import click
 
 from ..core import get_block_candidates, ModToolMakeYAML
 from ..tools import SequenceCompleter
 from .base import common_params, block_name, run, cli_input
 
+from gnuradio.blocktool import BlockHeaderParser
+from gnuradio.blocktool.cli.base import BlockToolException, run_blocktool
```

Comment:
there must not be any usage of the blocktool cli here - use the API of the blocktool package here not its CLI. pretend it doesn't exist. if that means moving stuff into core then that is what has to happen